

Starting Toontown Online...
Current time: Tue Dec 20 21:50:40 2022 Eastern Standard Time
sys.path =  ['', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python\\python27.zip', 'C:\\Panda3D-1.11.0-x64', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python\\DLLs', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python\\lib', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python\\lib\\plat-win', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python\\lib\\lib-tk', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\bin', 'C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\Panda3D-1.11.0-x64\\python\\lib\\site-packages']
sys.argv =  ['C:\\Toontown\\Toontown - Programming\\SuitopiaUpdateLastLaff\\Toontown-Last-Laff-Workspace\\src\\toontown\\launcher\\TTOffQuickStartLauncher.py']
generating standard configrc
os.environ =  {'TMP': 'C:\\Users\\theph\\AppData\\Local\\Temp', 'COMPUTERNAME': 'DESKTOP-7054VC9', 'PROCESSOR_LEVEL': '6', 'USERDOMAIN': 'DESKTOP-7054VC9', 'NEKO_INSTPATH': 'C:\\HaxeToolkit\\neko', 'INTEL_DEV_REDIST': 'C:\\Program Files (x86)\\Common Files\\Intel\\Shared Libraries\\', 'PSMODULEPATH': 'C:\\Program Files\\WindowsPowerShell\\Modules;C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules', 'COMMONPROGRAMFILES': 'C:\\Program Files\\Common Files', 'PROCESSOR_IDENTIFIER': 'Intel64 Family 6 Model 158 Stepping 13, GenuineIntel', 'PROGRAMFILES': 'C:\\Program Files', 'PROCESSOR_REVISION': '9e0d', 'PATH': 'C:\\Panda3D-1.11.0-x64\\panda3d\\..\\bin;C:\\Program Files (x86)\\Common Files\\Intel\\Shared Libraries\\redist\\intel64_win\\compiler;C:\\Program Files (x86)\\NVIDIA Corporation\\PhysX\\Common;C:\\Program Files\\Python38\\Scripts\\;C:\\Program Files\\Python38\\;C:\\WINDOWS\\system32;C:\\WINDOWS;C:\\WINDOWS\\System32\\Wbem;C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\;C:\\WINDOWS\\System32\\OpenSSH\\;C:\\Program Files\\NVIDIA Corporation\\NVIDIA NvDLISR;C:\\Program Files\\dotnet\\;C:\\Users\\theph\\AppData\\Local\\Programs\\Python\\Python39;C:\\Users\\theph\\AppData\\Local\\Programs\\Python\\Python39\\Scripts;C:\\Program Files\\Common Files\\Autodesk Shared\\;C:\\Program Files\\Git\\cmd;C:\\Program Files (x86)\\Windows Live\\Shared;C:\\HaxeToolkit\\haxe;C:\\HaxeToolkit\\neko;C:\\Program Files\\Microsoft SQL Server\\150\\Tools\\Binn\\;C:\\Program Files\\Microsoft SQL Server\\Client SDK\\ODBC\\170\\Tools\\Binn\\;C:\\Users\\theph\\AppData\\Local\\Microsoft\\WindowsApps;C:\\Users\\theph\\.dotnet\\tools;C:\\Program Files\\JetBrains\\PyCharm Community Edition 2020.3.3\\bin;C:\\Users\\theph\\AppData\\Local\\atom\\bin;C:\\Users\\theph\\AppData\\Local\\GitHubDesktop\\bin;C:\\Panda3D-1.9.0-x64\\python;C:\\Panda3D-1.9.0-x64\\python\\Scripts;C:\\Panda3D-1.9.0-x64\\bin;C:\\Panda3D-1.9.0\\python;C:\\Panda3D-1.9.0\\python\\Scripts;C:\\Panda3D-1.9.0\\bin;C:\\Panda3D-1.10.0\\python;C:\\Panda3D-1.10.0\\python\\Scripts;C:\\Panda3D-1.10.0\\bin;C:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.2.2\\bin;;C:\\Users\\theph\\AppData\\Local\\Programs\\Microsoft VS Code\\bin;C:\\Panda3D-1.11.0\\python;C:\\Panda3D-1.11.0\\python\\Scripts;C:\\Panda3D-1.11.0\\bin;C:\\Users\\theph\\.dotnet\\tools;C:\\Panda3D-1.10.11-x64\\python;C:\\Panda3D-1.10.11-x64\\python\\Scripts;C:\\Panda3D-1.10.11-x64\\bin;C:\\Panda3D-1.11.0-x64\\python;C:\\Panda3D-1.11.0-x64\\python\\Scripts;C:\\Panda3D-1.11.0-x64\\bin;C:\\EventHorizon-battle-ng-teaser\\dependencies\\panda\\python;C:\\EventHorizon-battle-ng-teaser\\dependencies\\panda\\python\\Scripts;C:\\EventHorizon-battle-ng-teaser\\depe', 'SYSTEMROOT': 'C:\\WINDOWS', 'PROGRAMFILES(X86)': 'C:\\Program Files (x86)', 'COMSPEC': 'C:\\WINDOWS\\system32\\cmd.exe', 'DRIVERDATA': 'C:\\Windows\\System32\\Drivers\\DriverData', 'PPYTHON_PATH': '"Panda3D-1.11.0-x64\\python\\ppython.exe"', 'TEMP': 'C:\\Users\\theph\\AppData\\Local\\Temp', 'FPS_BROWSER_APP_PROFILE_STRING': 'Internet Explorer', 'COMMONPROGRAMFILES(X86)': 'C:\\Program Files (x86)\\Common Files', 'PROCESSOR_ARCHITECTURE': 'AMD64', 'ALLUSERSPROFILE': 'C:\\ProgramData', 'USERPROFILE': 'C:\\Users\\theph', 'LOCALAPPDATA': 'C:\\Users\\theph\\AppData\\Local', 'FPS_BROWSER_USER_PROFILE_STRING': 'Default', 'HOMEPATH': '\\Users\\theph', 'USERDOMAIN_ROAMINGPROFILE': 'DESKTOP-7054VC9', 'PROGRAMW6432': 'C:\\Program Files', 'USERNAME': 'theph', 'LOGONSERVER': '\\\\DESKTOP-7054VC9', 'PROMPT': '$P$G', 'SESSIONNAME': 'Console', 'PROGRAMDATA': 'C:\\ProgramData', 'ONEDRIVE': 'C:\\Users\\theph\\OneDrive', 'ADSK_3DSMAX_X64_2019': 'C:\\Program Files\\Autodesk\\3ds Max 2019\\', 'ZES_ENABLE_SYSMAN': '1', 'HAXEPATH': 'C:\\HaxeToolkit\\haxe', 'PATHEXT': '.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW', 'TTOFF_LOGIN_TOKEN': 'dev', 'WINDIR': 'C:\\WINDOWS', 'PYCHARM COMMUNITY EDITION': 'C:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.2.2\\bin;', 'HOMEDRIVE': 'C:', 'SYSTEMDRIVE': 'C:', 'PUBLIC': 'C:\\Users\\Public', 'NUMBER_OF_PROCESSORS': '16', 'APPDATA': 'C:\\Users\\theph\\AppData\\Roaming', 'IDK2': 'C:\\Program Files\\Python38', 'COMMONPROGRAMW6432': 'C:\\Program Files\\Common Files', 'OS': 'Windows_NT', 'ADSK_CLM_WPAD_PROXY_CHECK': 'FALSE', 'IDK': 'C:\\Program Files\\Python38\\Scripts'}
:12-20-2022 21:50:40 TTOffLauncher: isTestServer: 0
:12-20-2022 21:50:40 TTOffLauncher: Game Server 127.0.0.1
:12-20-2022 21:50:40 TTOffLauncher: Starting Toontown Online... 
:12-20-2022 21:50:40 TTOffLauncher: foreground: Launcher now operating in foreground
useTTOffSpecificLogin=True
:12-20-2022 21:50:40 TTOffLauncher: maybeStartGame: starting game
:12-20-2022 21:50:40 TTOffLauncher: Starting Toontown Online...
:12-20-2022 21:50:40 TTOffLauncher: background: Launcher now operating in background
ToontownStart: Polling for game2 to finish...
ToontownStart: Game2 is finished.
ToontownStart: Starting the game.
ToontownStart: setting default font
:12-20-2022 21:50:40 TTOffLauncher: setting panda error code to 7
Known pipe types:
  wglGraphicsPipe
(all display modules loaded.)
:12-20-2022 21:50:40 ToonBase: Default graphics pipe is wglGraphicsPipe (OpenGL).
:12-20-2022 21:50:40 :display(warning): FrameBufferProperties available less than requested.
  requested: depth_bits=24 color_bits=3 red_bits=1 green_bits=1 blue_bits=1 alpha_bits=1 multisamples=70 back_buffers=1 force_hardware 
  got: depth_bits=24 color_bits=24 red_bits=8 green_bits=8 blue_bits=8 alpha_bits=8 accum_bits=64 multisamples=16 back_buffers=1 force_hardware 
:12-20-2022 21:50:41 ToonBase: Successfully opened window of type wglGraphicsWindow (OpenGL)
:12-20-2022 21:50:41 ShowBase: __dev__ == False
:12-20-2022 21:50:41 TTOffLauncher: setting panda error code to 11
:12-20-2022 21:50:41 TTOffLauncher: setting panda error code to 0
ToontownStart: Loading default gui sounds
ToontownStart: serverVersion:  tto-dev
:12-20-2022 21:50:41 Loader: starting bulk load of block 'init'
:12-20-2022 21:50:42 :nativenet(error): Buffered_DatagramConnection Constructor rbufsize = 4096000 wbufsize = 4096000 write_flush_point = 1400
:12-20-2022 21:50:42 OTPClientRepository: Using connect method 'http'
:12-20-2022 21:50:42 TTOffLauncher: getRegistry ('DEPLOYMENT', None)
:12-20-2022 21:50:42 TTOffLauncher: checking env
:12-20-2022 21:50:42 TTOffLauncher: getRegistry ('GAME_DEPLOYMENT', None)
:12-20-2022 21:50:42 TTOffLauncher: checking env
:12-20-2022 21:50:42 OTPClientRepository: reg_deployment=
:12-20-2022 21:50:42 OTPClientRepository: required-login auto.
:12-20-2022 21:50:42 OTPClientRepository: loginInterface: LoginTTOffAccount
:12-20-2022 21:50:42 TTOffLauncher: getNeedPwForSecretKey using ttoff-specific-login
:12-20-2022 21:50:42 Loader: At end of block 'init', expected 138, loaded 6, duration=1.3772139
:12-20-2022 21:50:42 OTPClientRepository: Connecting to gameserver directly (no proxy).
:12-20-2022 21:50:42 OTPClientRepository: Connecting to //127.0.0.1:7198 via HTTP interface.
:12-20-2022 21:50:42 TTOffLauncher: Stopping mini task manager.
:12-20-2022 21:50:42 OTPClientRepository: Successfully connected to //127.0.0.1:7198.
:12-20-2022 21:50:42 TTOffLauncher: Setting Disconnect Details normal
{402000001: <toontown.distributed.ToontownDistrict.ToontownDistrict instance at 0x0000000020739C48>}
:12-20-2022 21:50:45 StateData: AvatarChooser.enter
:12-20-2022 21:50:45 DisplayOptions: settingsFullScreen = False, embedded = False width=800 height=600
:12-20-2022 21:50:45 StateData: calling self.displayOptions.restrictToEmbedded(False)
:12-20-2022 21:51:20 OTPClientRepository: ================
:12-20-2022 21:51:20 OTPClientRepository: Chose avatar id: 100000001
:12-20-2022 21:51:20 OTPClientRepository: Chose avatar name: Prof. Alvin Flippenhopper
:12-20-2022 21:51:20 OTPClientRepository: Chose avatar dna: ('dls', 'ss', 's', 'm', 15, 0, 15, 15, 1, 8, 1, 8, 1, 1)
:12-20-2022 21:51:20 OTPClientRepository: Chose avatar position: 1
:12-20-2022 21:51:20 OTPClientRepository: isPaid: 1
:12-20-2022 21:51:20 OTPClientRepository: freeTimeLeft: 0
:12-20-2022 21:51:20 OTPClientRepository: allowSecretChat: True
:12-20-2022 21:51:20 OTPClientRepository: ================
:12-20-2022 21:51:20 Loader: starting bulk load of block 'localAvatarPlayGame'
:12-20-2022 21:51:20 ToontownTimeManager(warning): error parsing date string=
local
:12-20-2022 21:51:21 NewsPageButtonManager(warning): Should not get here. NewsPageButtonManager.exitOff
:12-20-2022 21:51:21 :pnmimage:png(warning): iCCP: known incorrect sRGB profile
:12-20-2022 21:51:22 OTPClientRepository: Sending heartbeat mid-frame.
:12-20-2022 21:51:24 OTPClientRepository: Entering shard 402000001
:12-20-2022 21:51:24 StreetSign: Street sign url is https://ttoffline.com/images/
:12-20-2022 21:51:25 TimeManager: Clock sync: TimeManager.announceGenerate
:12-20-2022 21:51:25 InGameNewsMgr: latestIssue=2013-08-22 16:49:46.000003-07:00
cpu info: GenuineIntel|Intel(R) Core(TM) i9-9900K CPU @ 3.60GHz|591597|0|3.600,3.600|8,16 cpus
:12-20-2022 21:51:26 OTPClientRepository: No sync from TimeManager.
:12-20-2022 21:51:26 OTPClientRepository: gotTimeSync
:12-20-2022 21:51:27 Loader: At end of block 'localAvatarPlayGame', expected 400, loaded 21, duration=6.5856738
:12-20-2022 21:51:27 TimeManager: Clock sync roundtrip took 1120.272 ms
:12-20-2022 21:51:27 TimeManager: AI time delta is -0:02 from server delta
:12-20-2022 21:51:27 TimeManager: Local clock uncertainty +/- 0.560 s
:12-20-2022 21:51:27 PlayGame: QA-REGRESSION: NEIGHBORHOODS: Visit Sellbot HQ
:12-20-2022 21:51:27 Loader: starting bulk load of block 'hood'
:12-20-2022 21:51:28 TownBattleSuitPanel: Loading Suit Battle Panel!
:12-20-2022 21:51:28 TownBattleSuitPanel: Loading Suit Battle Panel!
:12-20-2022 21:51:28 TownBattleSuitPanel: Loading Suit Battle Panel!
:12-20-2022 21:51:28 TownBattleSuitPanel: Loading Suit Battle Panel!
:12-20-2022 21:51:28 TownBattleSuitPanel: Loading Suit Battle Panel!
:12-20-2022 21:51:28 TownBattleSuitPanel: Loading Suit Battle Panel!
:12-20-2022 21:51:28 TownBattleSuitPanel: Loading Suit Battle Panel!
:12-20-2022 21:51:28 TownBattleSuitPanel: Loading Suit Battle Panel!
:12-20-2022 21:51:29 :pnmimage:png(warning): Interlace handling should be turned on when using png_read_image
:12-20-2022 21:51:29 SellbotCogHQLoader: loadPlaceGeom: 11000
:12-20-2022 21:51:29 DistributedObject: DistributedSuitPlanner 402002962: generating
:12-20-2022 21:51:29 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/nd.png
:12-20-2022 21:51:29 :gobj(error): Unable to find texture "phase_3/maps/hands/nd.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:12-20-2022 21:51:29 DistributedToon: calling LocalAvatar.startchat
:12-20-2022 21:51:29 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_sellbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:12-20-2022 21:51:30 Loader: At end of block 'hood', expected 43, loaded 14, duration=3.2259533
:12-20-2022 21:51:30 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/tf.png
:12-20-2022 21:51:30 :gobj(error): Unable to find texture "phase_3/maps/hands/tf.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:12-20-2022 21:51:30 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/tm.png
:12-20-2022 21:51:30 :gobj(error): Unable to find texture "phase_3/maps/hands/tm.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:12-20-2022 21:51:30 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/ms.png
:12-20-2022 21:51:30 :gobj(error): Unable to find texture "phase_3/maps/hands/ms.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:12-20-2022 21:51:30 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/tm.png
:12-20-2022 21:51:30 :gobj(error): Unable to find texture "phase_3/maps/hands/tm.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:12-20-2022 21:51:31 Teleport(debug): teleportInDone: walk
:12-20-2022 21:51:31 :loader(error): Couldn't load file phase_4/models/questmap/cog_hq_sellbot_maps.bam: not found on model path (currently: "/;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models")
:12-20-2022 21:51:31 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/ms.png
:12-20-2022 21:51:31 :gobj(error): Unable to find texture "phase_3/maps/hands/ms.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:12-20-2022 21:51:31 ClassicFSM(warning): [DistributedSuit]: no transition exists from Walk to FromCogHQ
:12-20-2022 21:51:32 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/tm.png
:12-20-2022 21:51:32 :gobj(error): Unable to find texture "phase_3/maps/hands/tm.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:12-20-2022 21:51:32 OTPClientRepository: Sending heartbeat mid-frame.
:12-20-2022 21:51:36 ToontownChatManager: QA-REGRESSION: CHAT: Speedchat Plus
:12-20-2022 21:51:43 :gobj(error): Texture::read() - couldn't read: phase_3/maps/hands/ms.png
:12-20-2022 21:51:43 :gobj(error): Unable to find texture "phase_3/maps/hands/ms.png" on model-path /;/c/Toontown/Toontown - Programming/SuitopiaUpdateLastLaff/Toontown-Last-Laff-Workspace/src/toontown/launcher;/c/Panda3D-1.11.0-x64/etc/..;/c/Panda3D-1.11.0-x64/etc/../models
Assertion failed: other._error_type == ET_ok at line 395 of panda/src/pgraph/nodePath.cxx
:12-20-2022 21:51:45 OTPClientRepository(warning): Server is booting us out (153): The session object with id 100000001 has been unexpectedly deleted.
:12-20-2022 21:51:45 OTPClientRepository(warning): Lost connection to gameserver.
:12-20-2022 21:51:45 SellbotHQExterior: Unloading Place (11000). Fsm in final
:12-20-2022 21:51:45 OTPClientRepository(warning): unclean exit, abandoning shard
:12-20-2022 21:51:45 DistributedObject: DistributedSuitPlanner 402002962: disabling
ignoring InputState-forward
:12-20-2022 21:51:45 OTPClientRepository: sending clientLogout
Assertion failed: _error_type != ET_not_found at line 592 of panda/src/pgraph/nodePath.cxx
Traceback (most recent call last):
  File "C:\Panda3D-1.11.0-x64\direct\distributed\ConnectionRepository.py", line 609, in readerPollUntilEmpty
    while self.readerPollOnce():
  File "C:\Panda3D-1.11.0-x64\direct\distributed\ConnectionRepository.py", line 616, in readerPollOnce
    self.handleDatagram(self.private__di)
  File "otp\distributed\OTPClientRepository.py", line 1838, in handleDatagram
    self.handler(msgType, di)
  File "otp\distributed\OTPClientRepository.py", line 1445, in handlePlayGame
    self.handleMessageType(msgType, di)
  File "otp\distributed\OTPClientRepository.py", line 1716, in handleMessageType
    self.handleGoGetLost(di)
  File "C:\Panda3D-1.11.0-x64\direct\distributed\ClientRepositoryBase.py", line 494, in handleGoGetLost
    self.lostConnection()
  File "otp\distributed\OTPClientRepository.py", line 1621, in lostConnection
    self.loginFSM.request('noConnection')
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 281, in request
    exitArgList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 227, in __transition
    self.__exitCurrent(exitArgList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 185, in __exitCurrent
    self.__currentState.exit(argList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\State.py", line 213, in exit
    self.__exitFunc(*argList)
  File "toontown\distributed\ToontownClientRepository.py", line 430, in exitPlayingGame
    base.localAvatar.deleteOrDelay()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 201, in deleteOrDelay
    self.disableAnnounceAndDelete()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 204, in disableAnnounceAndDelete
    self.disableAndAnnounce()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 230, in disableAndAnnounce
    self.disable()
  File "toontown\toon\LocalToon.py", line 256, in disable
    self.book.unload()
  File "toontown\shtiker\ShtikerBook.py", line 150, in unload
    page.unload()
  File "toontown\shtiker\SuitPage.py", line 240, in unload
    shadow.removeNode()
AssertionError: _error_type != ET_not_found at line 592 of panda/src/pgraph/nodePath.cxx
:12-20-2022 21:51:45 :task(error): Exception occurred in PythonTask readerPollTask-142969536
Traceback (most recent call last):
  File "C:\Panda3D-1.11.0-x64\direct\showbase\EventManager.py", line 49, in eventLoopTask
    self.doEvents()
  File "C:\Panda3D-1.11.0-x64\direct\showbase\EventManager.py", line 42, in doEvents
    while not isEmptyFunc():
  File "C:\Panda3D-1.11.0-x64\direct\distributed\ConnectionRepository.py", line 609, in readerPollUntilEmpty
    while self.readerPollOnce():
  File "C:\Panda3D-1.11.0-x64\direct\distributed\ConnectionRepository.py", line 616, in readerPollOnce
    self.handleDatagram(self.private__di)
  File "otp\distributed\OTPClientRepository.py", line 1838, in handleDatagram
    self.handler(msgType, di)
  File "otp\distributed\OTPClientRepository.py", line 1445, in handlePlayGame
    self.handleMessageType(msgType, di)
  File "otp\distributed\OTPClientRepository.py", line 1716, in handleMessageType
    self.handleGoGetLost(di)
  File "C:\Panda3D-1.11.0-x64\direct\distributed\ClientRepositoryBase.py", line 494, in handleGoGetLost
    self.lostConnection()
  File "otp\distributed\OTPClientRepository.py", line 1621, in lostConnection
    self.loginFSM.request('noConnection')
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 281, in request
    exitArgList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 227, in __transition
    self.__exitCurrent(exitArgList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 185, in __exitCurrent
    self.__currentState.exit(argList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\State.py", line 213, in exit
    self.__exitFunc(*argList)
  File "toontown\distributed\ToontownClientRepository.py", line 430, in exitPlayingGame
    base.localAvatar.deleteOrDelay()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 201, in deleteOrDelay
    self.disableAnnounceAndDelete()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 204, in disableAnnounceAndDelete
    self.disableAndAnnounce()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 230, in disableAndAnnounce
    self.disable()
  File "toontown\toon\LocalToon.py", line 256, in disable
    self.book.unload()
  File "toontown\shtiker\ShtikerBook.py", line 150, in unload
    page.unload()
  File "toontown\shtiker\SuitPage.py", line 240, in unload
    shadow.removeNode()
AssertionError: _error_type != ET_not_found at line 592 of panda/src/pgraph/nodePath.cxx
:12-20-2022 21:51:45 :task(error): Exception occurred in PythonTask eventManager
:12-20-2022 21:51:45 TTOffLauncher: setting panda error code to 12
:12-20-2022 21:51:45 TTOffLauncher(warning): Handling Python exception.
:12-20-2022 21:51:45 OTPClientRepository: Sent disconnect message to server
:12-20-2022 21:51:45 OTPClientRepository: Closing connection to server.
:12-20-2022 21:51:45 TTOffLauncher: Exception exit.

Traceback (most recent call last):
  File "otp\launcher\LauncherBase.py", line 1081, in mainLoop
    self._runTaskManager()
  File "otp\launcher\LauncherBase.py", line 1052, in _runTaskManager
    taskMgr.run()
  File "C:\Panda3D-1.11.0-x64\direct\task\Task.py", line 528, in run
    self.step()
  File "C:\Panda3D-1.11.0-x64\direct\task\Task.py", line 482, in step
    self.mgr.poll()
  File "C:\Panda3D-1.11.0-x64\direct\showbase\EventManager.py", line 49, in eventLoopTask
    self.doEvents()
  File "C:\Panda3D-1.11.0-x64\direct\showbase\EventManager.py", line 42, in doEvents
    while not isEmptyFunc():
  File "C:\Panda3D-1.11.0-x64\direct\distributed\ConnectionRepository.py", line 609, in readerPollUntilEmpty
    while self.readerPollOnce():
  File "C:\Panda3D-1.11.0-x64\direct\distributed\ConnectionRepository.py", line 616, in readerPollOnce
    self.handleDatagram(self.private__di)
  File "otp\distributed\OTPClientRepository.py", line 1838, in handleDatagram
    self.handler(msgType, di)
  File "otp\distributed\OTPClientRepository.py", line 1445, in handlePlayGame
    self.handleMessageType(msgType, di)
  File "otp\distributed\OTPClientRepository.py", line 1716, in handleMessageType
    self.handleGoGetLost(di)
  File "C:\Panda3D-1.11.0-x64\direct\distributed\ClientRepositoryBase.py", line 494, in handleGoGetLost
    self.lostConnection()
  File "otp\distributed\OTPClientRepository.py", line 1621, in lostConnection
    self.loginFSM.request('noConnection')
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 281, in request
    exitArgList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 227, in __transition
    self.__exitCurrent(exitArgList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\ClassicFSM.py", line 185, in __exitCurrent
    self.__currentState.exit(argList)
  File "C:\Panda3D-1.11.0-x64\direct\fsm\State.py", line 213, in exit
    self.__exitFunc(*argList)
  File "toontown\distributed\ToontownClientRepository.py", line 430, in exitPlayingGame
    base.localAvatar.deleteOrDelay()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 201, in deleteOrDelay
    self.disableAnnounceAndDelete()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 204, in disableAnnounceAndDelete
    self.disableAndAnnounce()
  File "C:\Panda3D-1.11.0-x64\direct\distributed\DistributedObject.py", line 230, in disableAndAnnounce
    self.disable()
  File "toontown\toon\LocalToon.py", line 256, in disable
    self.book.unload()
  File "toontown\shtiker\ShtikerBook.py", line 150, in unload
    page.unload()
  File "toontown\shtiker\SuitPage.py", line 240, in unload
    shadow.removeNode()
AssertionError: _error_type != ET_not_found at line 592 of panda/src/pgraph/nodePath.cxx
